packages/containers/src/Form/Form.test.js

```diff
@@ -39,37 +37,31 @@ describe('Container(Form)', () => {
 	it('should render with prop uiform = false : Form', () => {
 		const wrapper = mount(<Container jsonSchema={{}} uiSchema={{}} uiform={false} />);
 
-		expect(wrapper.find(Form).length).toBe(1);
-		expect(wrapper.find(UIForm).length).toBe(0);
+		expect(wrapper.find('TalendForm').length).toBe(1);
+		expect(wrapper.find('TalendUIForm').length).toBe(0);
 	});
 
 	it('should render with prop uiform = true : UIForm', () => {
 		const wrapper = mount(<Container jsonSchema={{}} uiSchema={[]} uiform />);
-		expect(wrapper.find(Form).length).toBe(1);
-		expect(wrapper.find(UIForm).length).toBe(1);
+		expect(wrapper.find('TalendForm').length).toBe(1);
+		expect(wrapper.find('TalendUIForm').length).toBe(1);
 	});
 
 	it('should render UIForm with language prop set', () => {
-		const wrapper = shallow(
+		const wrapper = mount(
 			<Container
 				formId="test-form"
-				jsonSchema={{ schema: true }}
-				uiSchema={{ uiSchema: true }}
+				jsonSchema={{}}
+				uiSchema={{}}
 				actions={[]}
-				formProps={{ other: true }} // extra props
+				formProps={{ other: true }}
 				uiform
 				language={{ OBJECT_REQUIRED: 'Field translated' }}
 			/>,
 		);
-		expect(
-			wrapper
-				.find('TalendForm')
-				.dive()
-				.find('Container(UIForm)')
-				.dive()
-				.find('TalendUIForm')
-				.props().language.OBJECT_REQUIRED,
-		).toEqual('Field translated');
+		const wrapperTalendForm = wrapper.find('TalendForm');
+		const wrapperTalendUIForm = wrapperTalendForm.find('TalendUIForm');
```

Comment:
You don't need to get TalendForm then TalendUIForm. You can directly write `wrapper.find('TalendUIForm')`